shell/platform/fuchsia/flutter/flatland_connection.h

```diff
@@ -137,7 +137,7 @@ class FlatlandConnection final {
     fml::TimePoint last_presentation_time_;
     FireCallbackCallback pending_fire_callback_;
     uint32_t present_credits_ = 1;
-    bool first_feedback_received_ = false;
+    bool maybe_run_initial_vsync_callback_sent_ = false;
```

Comment:
The name suggests it should be true. What about renaming it "initial_vsync_callback_ran_ = false"?

Reply:
Done